README.md

```diff
@@ -376,18 +376,27 @@ This repository contains a curated list of awesome open source libraries that wi
 * [CuML](https://github.com/rapidsai/cuml) ![](https://img.shields.io/github/stars/rapidsai/cuml.svg?style=social) - cuML is a suite of libraries that implement machine learning algorithms and mathematical primitives functions that share compatible APIs with other RAPIDS projects.
 * [CuPy](https://github.com/cupy/cupy) ![](https://img.shields.io/github/stars/cupy/cupy.svg?style=social) - An implementation of NumPy-compatible multi-dimensional array on CUDA. CuPy consists of the core multi-dimensional array class, cupy.ndarray, and many functions on it.
 * [DeepSpeed MII](https://github.com/microsoft/DeepSpeed-MII) ![](https://img.shields.io/github/stars/microsoft/DeepSpeed-MII.svg?style=social) - DeepSpeed MII makes low-latency and high-throughput inference possible, powered by DeepSpeed.
+* [fastai](https://github.com/fastai/fastai) ![](https://img.shields.io/github/stars/fastai/fastai.svg?style=social) - fastai is a deep learning library which provides practitioners with high-level components that can quickly and easily provide state-of-the-art results in standard deep learning domains, and provides researchers with low-level components that can be mixed and matched to build new approaches.
```

Comment:
I'm not sure I would describe fastai as computation optimization, more than a tooling to streamline DS/MLE workflows

Reply:
Hi @axsaucedo I have used fastai before and this library is popular for its convenience to use, you can check the relevant info below:
![image](https://user-images.githubusercontent.com/8592144/222985622-909e915d-77d5-4a57-bb10-be0a478556e1.png)
![image](https://user-images.githubusercontent.com/8592144/222985647-6a58f771-6c0f-425e-9085-21ac997d3be0.png)

Reply:
Yeah i've used fastai as well, and it's great, just not productionisation framework, more of library to perform ML experimentation workflows - hence why keras is also not on the list. It seems fastai has evolved a lot since I last used tho, does it have any productionisation workflows as opposed to experimentation tooling? I'm still not certain of computation optimisation...

Reply:
> 

Good question, I did not include Keras due to the facts that Keras is already fully integrated into Tensorflow, we just need to include Tensorflow, then we never bothers to include Keras at all.

I know some DS use fastai to do fast prototyping in companies, so I think fastai has a reason to be kept on the list. Maybe we should keep it somewhere in the list? @axsaucedo

Reply:
> I know some DS use fastai to do fast prototyping in companies
fast prototyping is indeed useful but not for productionising, let's please remove so we can merge

Reply:
> 

I google it for the concept between "prototyping" and "production" and indeed found the former belongs to the category of the latter. 
![image](https://user-images.githubusercontent.com/8592144/223328251-803180f6-f859-4ba2-b022-39450f72bfe4.png)
If a tool help practitioners to achieve their product goals then it is already used in real-world production. @axsaucedo

Reply:
We're going too much back and forth on terminology, prototyping and produciton is pretty much an oxymoron so if the argument is let's add because it's good for prototyping let's definitley remove - at this stage i don't see this as comptuation optimisation tool, given PR is to add tools on this section I don't see this as relevant for this pr

Reply:
The reason why we are back and forth is that you did not give an exact definition of production machine learning in CONTRIBUTE.md. I do think it is high time to give us an explanation, or I still cannot understand why these tools should be excluded. https://github.com/EthicalML/awesome-production-machine-learning/issues/392 @axsaucedo

Reply:
Unfortunately when it comes to maintenance of OSS libraries there needs to be an ultimate decision that needs to be made, we can end up discussing forever but when it comes to adding 1-2 lines we can't have endless discussions. The end decisions won't be perfect, and there will be ambiguity as we won't reach terminology that will be exact without overlap, and there will also be subjectivity as this list is maintained by humans.

However we can't be discussing forever and a decision has to be made, unfortunately if a tool is being added or pushed back on, people may agree or disagree but we'll have to commit and move forward - I am aware you can't keep everyone happy but that's what happens in larger projects. 

This is to manage expectations as we could try to put together a definition of "production ML" in the contribute and that will still not provide the full non-overlapping clarity.

Because of this I want to manage expectations as there will be future decisions in this and many other projects that may not be perfect but that need to be made in order to progress.